tests/AbstractSnowflakeTest.php

```diff
@@ -241,10 +253,10 @@ protected function createTextTable(CsvFile $file, $tableName = null, $schemaName
     /**
      * Count records in CSV (with headers)
      *
-     * @param CsvFile $file
+     * @param  CsvFile $file
```

Comment:
It seems you need to disable attribute alignment in you phpstorm code style settings... let me know if you couldn't find it.